src/azure-cli/azure/cli/command_modules/iot/custom.py

```diff
@@ -1416,6 +1416,114 @@ def _get_iot_central_app_by_name(client, app_name):
 
 
+def iot_central_list_private_link_resource(client, app_name=None, connection_id=None, resource_group_name=None, source_type=None):
+    if app_name and resource_group_name and source_type:
+        if source_type != 'Microsoft.IoTCentral/iotApps':
+            raise CLIError(
+                "Type must be Microsoft.IoTCentral/iotApps")
+    elif connection_id:
+        id_list = connection_id.split('/')
+        resource_group_name = id_list[id_list.index('resourceGroups') + 1]
+        app_name = id_list[id_list.index('iotApps') + 1]
+    else:
+        raise CLIError(
+            "Must provide id or application name, resource group, and type.")
+    return client.private_links.list(resource_group_name, app_name)
+
+
+def show_private_endpoint_connection(client, resource_group_name=None, connection_id=None, account_name=None, private_endpoint_connection_name=None):
+
+    return get_private_endpoint_connection(client=client,
+                                           resource_group_name=resource_group_name,
+                                           connection_id=connection_id,
+                                           account_name=account_name,
+                                           private_endpoint_connection_name=private_endpoint_connection_name,
+                                           return_args=False)
+
+
+def get_private_endpoint_connection(client, resource_group_name=None, connection_id=None, account_name=None, private_endpoint_connection_name=None, return_args=False):
+
+    if resource_group_name and account_name and private_endpoint_connection_name:
+        output = client.private_endpoint_connections.get(resource_group_name=resource_group_name,
+                                                         resource_name=account_name,
+                                                         private_endpoint_connection_name=private_endpoint_connection_name)
+        if return_args is False:
+            return output
+        return [output, resource_group_name, account_name, private_endpoint_connection_name]
+    if connection_id:
+        id_list = connection_id.split('/')
+        resource_group_name = id_list[id_list.index('resourceGroups') + 1]
+        account_name = id_list[id_list.index('iotApps') + 1]
+        private_endpoint_connection_name = id_list[id_list.index('privateEndpointConnections') + 1]
+        output = client.private_endpoint_connections.get(resource_group_name=resource_group_name,
+                                                         resource_name=account_name,
+                                                         private_endpoint_connection_name=private_endpoint_connection_name)
+        if return_args is False:
+            return output
+        return [output, resource_group_name, account_name, private_endpoint_connection_name]
+    raise CLIError(
+        "Account name, resource group, and private endpoint connection name are required unless id is specified.")
```

Comment:
Could you please use the specific error type `RequiredArgumentMissingError` instead of `CLIError`?